samples/config/minimal_config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "enclave_file": "lib_jsgeneric.enclave.so.signed",
+  "enclave_type": "release",
+  "network": {
+    "node_address": { "hostname": "127.0.0.1", "port": "0" },
```

Comment:
Merge "hostname" and "port".

Reply:
Will do in a follow-up PR: https://github.com/microsoft/CCF/issues/2612#issuecomment-973067527